single-function/add-two-numbers/02-impl-revision.js

```diff
@@ -1,24 +1,24 @@
 const addTwoNumbers = (l1, l2) => {
-  let carryover = 0,
-    dummyhead = { val: 0, next: null },
-    curr = dummyhead;
+  let carryOver = 0;
+  const dummyHead = { val: 0, next: null };
+  let curr = dummyHead;
 
   while (l1 !== null || l2 !== null) {
```

Comment:
Maybe more idiomatic in JS to say:

```
while (list1 || list2) {
```

No biggie, just a bit cleaner and it's common.  AirBnb style guide is fine with it: https://github.com/airbnb/javascript#comparison--if

Reply:
yeah much cleaner